dpe/src/dpe_instance.rs

```diff
@@ -296,7 +296,10 @@ impl<C: Crypto, P: Platform> DpeInstance<'_, C, P> {
             let context = status?;
 
             let mut tci_bytes = [0u8; size_of::<TciNodeData>()];
-            let len = context.tci.serialize(&mut tci_bytes)?;
+            let len = match context.tci.serialize(&mut tci_bytes) {
+                Ok(usize) => Ok(usize),
+                Err(_) => Err(DpeErrorCode::InvalidArgument)
+            }?;
```

Comment:
fyi: this can be simplified to
```suggestion
            let len = match context.tci.serialize(&mut tci_bytes).map_err(|_| DpeErrorCode::InvalidArgument)?;
```

Reply:
A lot of features to learn in rust :) thank you!

I changed this in 3 places.